packages/manager/modules/pci/src/projects/project/storages/cold-archive/users/import/import.controller.js

```diff
@@ -75,10 +73,17 @@ export default class PciBlockStorageContainersContainerObjectAddController {
   }
 
   cancel() {
-    this.atInternet.trackClick({
-      name: `${this.trackingPrefix}${TRACKING.IMPORT_POLICY}::cancel`,
-      type: 'action',
-    });
+    this.trackImportModalClick(COLD_ARCHIVE_TRACKING.ACTIONS.CANCEL);
     return this.goBack();
   }
+
+  trackImportModalPage(action) {
+    const hit = `${COLD_ARCHIVE_TRACKING.USER.MAIN}::${COLD_ARCHIVE_TRACKING.USER.ACTIONS.IMPORT_POLICY}_${action}`;
```

Comment:
for readability, may be better to have this:
```suggestion
    const {MAIN, ACTIONS} =  COLD_ARCHIVE_TRACKING.USER;
    const hit = `${MAIN}::${ACTIONS.IMPORT_POLICY}_${action}`;
```